docs/auth/account.rst

```diff
@@ -168,12 +168,10 @@ If email address and password match our records, the server will reply with
 
 As indicated in the response, login tokens expire 7 days after creation or
 when not used for 1 hour, whichever comes first (see :ref:`token object`).
+Expired login tokens are purged on each new login.
```

Comment:
The login tokens that we keep is the closest thing to an audit log that we can offer. Should we really be deleting it unconditionally? Alternatives:

- offer an endpoint to remove them
- offer a user setting that controls if they are automatically deleted
- do not display them by default when querying the tokens endpoint

Reply:
I also doubt people would use this as an audit log, as there is a lack of understanding about these tokens in the first place. We've received a couple (10?) support request over time where people expressed concerns their accounts had been hacked and those have been created etc. An audit log would create transparency, not confusion. :)

Reply:
Agree that it conflates concepts. Agree with user demand for deletion.

The audit comment wasn't aimed at users but at admins of deployments (specifically, you :) ).